statement_vtab.c

```diff
@@ -278,7 +278,7 @@ static sqlite3_module statement_vtab_module = {
 	.xRowid      = statement_vtab_rowid,
 };
 
-int sqlite3_extension_init(sqlite3* db, char** pzErrMsg, const sqlite3_api_routines* pApi) {
+int sqlite3_stmtvtab_init(sqlite3* db, char** pzErrMsg, const sqlite3_api_routines* pApi) {
```

Comment:
Makes sense, but can this be changed to `sqlite3_statementvtab_init`? This allows `load_extension` to continue to work without needing to specify the entry point according to https://www.sqlite.org/loadext.html

Reply:
Thanks for your feedback.. updated.